lib/services/android-plugin-build-service.ts

```diff
@@ -294,26 +294,55 @@ export class AndroidPluginBuildService implements IAndroidPluginBuildService {
 	}
 
 	private async getRuntimeGradleVersions(projectDir: string): Promise<IRuntimeGradleVersions> {
-		const registryData = await this.$packageManager.getRegistryPackageData(TNS_ANDROID_RUNTIME_NAME);
 		let runtimeGradleVersions: IRuntimeGradleVersions = null;
 		if (projectDir) {
 			const projectRuntimeVersion = this.$platformService.getCurrentPlatformVersion(
 				this.$devicePlatformsConstants.Android,
 				this.$projectDataService.getProjectData(projectDir));
-			runtimeGradleVersions = this.getGradleVersions(registryData.versions[projectRuntimeVersion]);
+			runtimeGradleVersions = await this.getGradleVersions(projectRuntimeVersion);
 			this.$logger.trace(`Got gradle versions ${JSON.stringify(runtimeGradleVersions)} from runtime v${projectRuntimeVersion}`);
 		}
 
 		if (!runtimeGradleVersions) {
-			const latestRuntimeVersion = registryData["dist-tags"].latest;
-			runtimeGradleVersions = this.getGradleVersions(registryData.versions[latestRuntimeVersion]);
+			const latestRuntimeVersion = await this.getLatestRuntimeVersion();
+			runtimeGradleVersions = await this.getGradleVersions(latestRuntimeVersion);
 			this.$logger.trace(`Got gradle versions ${JSON.stringify(runtimeGradleVersions)} from the latest runtime v${latestRuntimeVersion}`);
 		}
 
 		return runtimeGradleVersions || {};
 	}
 
-	private getGradleVersions(packageData: { gradle: { version: string, android: string } }): IRuntimeGradleVersions {
+	private async getLatestRuntimeVersion(): Promise<string> {
+		let runtimeVersion: string = null;
+
+		try {
+			const result = await this.$packageManager.view(TNS_ANDROID_RUNTIME_NAME, { "dist-tags": true });
+			runtimeVersion = result.latest;
+		} catch (err) {
+			this.$logger.trace('error while getting latest android runtime version from view command');
```

Comment:
Please print the real error today.
Also, `Error` should start with capital letter.